source/slang/slang-check-impl.h

```diff
@@ -2057,9 +2057,14 @@ namespace Slang
             ///
         SubtypeWitness* isSubtype(
             Type*                   subType,
-            Type*                   superType);
+            Type*                   superType,
+            bool subtypeInheritanceIsNotFullyResolved = false //resolving members may not be required in `SemanticsDeclBasesVisitor`
```

Comment:
Please avoid unnecessary default function values.
Also move the comment to the doxygen comments placed above the function name.